src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -280,6 +289,64 @@ public void shutdown(ExecutorDriver d) {
     log.info("Executor asked to shutdown");
   }
 
+  protected void runHealthChecks(ExecutorDriver driver, Task task) {
+    log.info("Performing health check for task: " + task.taskInfo.getTaskId().getValue());
+    boolean taskHealthy = false;
+    String taskIdStr = task.taskInfo.getTaskId().getValue();
```

Comment:
Why not get this variable before the log statement above, so you can reuse it there?